### ISSUE TITLE
Inconsistent toolbar button positioning

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-google-docs-integration#63]
On different reloads in Chrome:

![Screen Shot 2022-05-23 at 3 42 10 AM](https://user-images.githubusercontent.com/783082/169768738-1f2548f9-4d8d-464d-ad20-dcfbd4a0c24c.png)

![Screen Shot 2022-05-23 at 3 42 00 AM](https://user-images.githubusercontent.com/783082/169768725-22e3d1d4-1546-488d-aa6c-50c78e379391.png)

In the past I've seen it much further to the right (which seemed appropriate), but I can't get that to happen now. Can we control the positioning?

**Comment by dstillman** [comment]
(And, needless to say, appearing in either of these places is bad. If we can't control this anymore, we should get rid of the button.)

**Comment by adomasven** [comment]
Yeah, the button started moving recently although not as far to the left. I looked into it and it seems that most of the gdocs buttons are loaded dynamically now. I'll see if I can move the button and keep it from getting unbound from Google Docs management (i.e. automatically folding under `...` menu when resizing) and if not we'll have to remove it.
